be/src/service/service_be/starrocks_be.cpp

```diff
@@ -201,6 +103,10 @@ void start_be(const std::vector<StorePath>& paths, bool as_cn) {
     CHECK(global_vars->is_init()) << "global variables not initialized";
     LOG(INFO) << process_name << " start step " << start_step++ << ": global variables init successfully";
 
+    auto* cache_env = CacheEnv::GetInstance();
+    EXIT_IF_ERROR(cache_env->init(paths));
+    LOG(INFO) << process_name << " start step " << start_step++ << ": cache env init successfully";
+
```

Comment:
Do you think it's better to init the datacache after the storage engine? 
Because we depend on the storage paths, and the storage paths will be checked when initializing storage engine. If they are not existed or has some permission problem, it can be reported before we initializing datacache.

Reply:
Fixed